js/package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
```

Comment:
`L = ENG-71` -> `L=ENG-71`

You also have a squashed temp to remove from your commit message.